wordle-web/pages/game.vue

```diff
@@ -12,15 +12,17 @@
     <v-card-text class="text-h1 font-weight-black text-center">
       Wordle!
     </v-card-text>
-
+    
     <v-alert v-if="wordleGame.gameOver" width="80%" :type="gameResult.type">
       {{ gameResult.text }}
       <v-btn class="ml-2" @click="resetGame"> Play Again? </v-btn>
+      <v-btn class="ml-2" @click="savePlayerScore"> Save Game </v-btn>
```

Comment:
This should automatically prompt when the game is over, rather than having the user click it manually.